git_machete/github.py

```diff
@@ -131,6 +139,14 @@ def get_token_from_file_in_home_directory() -> Optional[str]:
             get_token_from_hub())
 
 
+def __get_github_token() -> Optional[str]:
+    return __get_github_token_and_provider().token if __get_github_token_and_provider() else None
```

Comment:
Nit: I'd extract the result of `__get_github_token_and_provider()` to a variable to avoid a double call